### PR TITLE
Add stablecoins mint and burn tables

### PR DESCRIPTION
**Pull request by milkyklim**
I've checked that:

* [ ] the query produces the intended results
* [ ] the folder name matches the schema name
* [ ] the schema name exists in Dune
* [ ] views are prefixed with `view_`, functions with `fn_`.
* [ ] the filename matches the defined view, table or function and ends with .sql
* [ ] each file has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`

### CODE REVIEW

File: schema/stablecoin/mint.sql [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    token_amount numeric,
+    usd_value numeric
+);
+
+CREATE OR REPLACE FUNCTION stablecoin.insert_mint(start_ts timestamptz, end_ts timestamptz=now(), start_block numeric=0, end_block numeric=9e18) RETURNS integer
+LANGUAGE plpgsql AS $function$
+DECLARE r integer;
+BEGIN
+WITH mint AS (
+    SELECT "name" as project,
```
Comment (milkyklim) [inline schema/stablecoin/mint.sql:22]: Using stablecoin `name` as project.

File: schema/stablecoin/mint.sql [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+           "version",
```
Comment (milkyklim) [inline schema/stablecoin/mint.sql:23]: Is everything except Maker `version` 1?

File: schema/stablecoin/mint.sql [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+           evt_block_number AS block_number,
+           evt_tx_hash AS tx_hash,
+           evt_index,
+           trace_address,
+           "to" AS minter,
+           "from" AS tx_from,
+           st.contract_address AS asset_address,
+           st.symbol AS asset_symbol,
+           value / 10^st.decimals AS token_amount,
+           value / 10^st.decimals * p.price AS usd_value
```
Comment (milkyklim) [inline schema/stablecoin/mint.sql:34]: Is price necessary for stables?

Comment (mewwts) [reply schema/stablecoin/mint.sql:34]: @hagaetc

File: schema/stablecoin/mint.sql (hunk 2 above)
Comment (mewwts) [reply schema/stablecoin/mint.sql:23]: @hagaetc

File: schema/stablecoin/mint.sql (hunk 1 above)
Comment (mewwts) [reply schema/stablecoin/mint.sql:22]: @hagaetc

**Review by mewwts — CHANGES_REQUESTED** [review]

File: schema/stablecoin/burn.sql [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+       amount_raw,
+       evt_index,
+       trace_address
+    )
+    SELECT
+       "name",
+       block_time,
+       tx_hash,
+       burner,
+       from_address,
```
[the remark this answers is not in the document]
Comment (hagaetc) [reply schema/stablecoin/burn.sql:103]: Should be `to_address`

**Comment by mewwts** [comment]
merged here https://github.com/duneanalytics/abstractions/commit/0a9bef540441035e09fcd762dcb30d24fe44b578

**Comment by mewwts** [comment]
Thanks @milkyklim

**Pull request closed without merging**